content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -772,8 +774,9 @@ validations are not supported by ratcheting under the implementation in Kubernet
   To remove a previously specified `additionalProperties` validation will not be
   ratcheted.
 - `metadata`
-  Errors arising from changes to fields within an object's `metadata` are not
-  ratcheted.
+  Errors arising Kubernetes' built-in validation of an object's `metadata` are not
+  ratcheted (such as name, label format, etc.), but user=sepcified rules on metadata 
```

Comment:
We [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations.
```suggestion
  ratcheted (such as object name, or characters in a label value). If you specify your own additional rules
  for the metadata of a custom resource, that additional validation
```

Reply:
The linked doc mentions an exception for etc., is it okay to use that here?

> Exception: Use "etc." for et cetera

Reply:
D'oh. That one is OK. I'd still fix the typo (“sepcified”) and reword along the lines of my suggestion, though.